src/domain/noteSettings.service.ts

```diff
@@ -0,0 +1,39 @@
+import type NoteSettingsRepository from '@/domain/noteSettings.repository.interface';
+import type NotesSettings from '@/domain/entities/NotesSettings';
+import type { NoteId } from './entities/Note';
+
+/**
+ * Note Service
+ */
+export default class NoteService {
+  /**
+   * Note repository
+   */
+  private readonly noteSettingsRepository: NoteSettingsRepository;
+
+  /**
+   * Note Service constructor
+   *
+   * @param noteSettingsRepository - Note repository instance
+   */
+  constructor(noteSettingsRepository: NoteSettingsRepository) {
+    this.noteSettingsRepository = noteSettingsRepository;
+  }
+
+  /**
+   * Get notesSettings
+   *
+   * @param id - Note id
+   */
+  public async getNotesSettingsById(id: NoteId): Promise<NotesSettings | null> {
+    let result;
+
+    try {
+      result = await this.noteSettingsRepository.getNotesSettingsById(id);
+    } catch (error) {
+      throw new Error(`404 Not Found: We couldn't find ${id} note.`);
+    }
```

Comment:
I think, error handling and ui message should be at the presentation layer. Also you need to check the error for `NotFoundError` instance. 

Rough example:

```ts
/**
 * Get note settings
 *
 * @param id - Note id
 */
 const load = async (id: NoteId): Promise<void> => {
   try { 
     noteSettings.value = await noteSettingsService.getNotesSettingsById(id);
   } catch (error){
     if (error instanceof NotFoundError) {
       router.replace({
         name: "404"
       });
       
       return;
     }
     
     // other errors should be thrown down
     throw error;
   }
 };
```

I think error handling could be done via a separate PR.